org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WPAttributeDialog.java

```diff
@@ -1246,4 +1259,29 @@ public boolean isChanged()
 		return m_changed;
 	}	//	isChanged
 
+	/**
+	 * This method searches for User's Window Access to determinate if it is
+	 * possible to create new ASI records (when IsReadWrite = true), only read
+	 * existing ASI records (when IsReadWrite = false) or open the ASI dialog (when
+	 * there is no Window Access for Attribute Set Instance window).
+	 */
+	private void validadeRoleAccess() {
+		List<MWindowAccess> windowAccesses = new Query(Env.getCtx(), MWindowAccess.Table_Name, "AD_Role_ID = ? AND AD_Window_ID = ?", null)
+				.setParameters(Env.getAD_Role_ID(Env.getCtx()), ATTRIBUTE_SET_INSTANCE_AD_Window_ID)
+				.setClient_ID()
+				.setOnlyActiveRecords(true)
+				.list();
+
+		if (windowAccesses.size() == 0)
+			throw new AdempiereException(Msg.translate(Env.getCtx(), "AccessTableNoView"));
+
+		for (MWindowAccess access : windowAccesses) {
+
+			if (access.isReadWrite()) {
+				isAllowedToCreate = true;
+				break;
+			}
+		}

```

Comment:
this is wrong, should just use `MRole.getDefault().getWindowAccess(ATTRIBUTE_SET_INSTANCE_AD_Window_ID)`